_posts/2016-05-30-one-model-to-rule-them-all.md

```diff
@@ -0,0 +1,154 @@
+---
+layout:      post
+title:       "One model to rule them all: cross-platform mobile test automation at BlaBlaCar"
```

Comment:
Just a nit, but English grammar rules usually want us to capitalize like: "One Model to Rule Them All: Cross-platform Mobile Test Automation at BlaBlaCar"--can you change it?